contracts/bonding/BondingManager.sol

```diff
@@ -71,22 +91,23 @@ contract BondingManager is IBondingManager, Controllable {
     // Mapping to track the index position of an address in the current active transcoder set
     mapping (address => uint256) public activeTranscoderPositions;
 
-    // Mapping to track transcoder's reward multiplier for a round
-    // rewardMultiplier[0] -> total delegator token rewards for a round (minus transcoder share)
-    // rewardMultiplier[1] -> transcoder's cumulative stake for round
-    mapping (address => mapping (uint256 => uint256[2])) public rewardMultiplierPerTranscoderAndRound;
-
     // Only the RoundsManager can call
-    modifier roundsManagerOnly() {
-        if (msg.sender != address(roundsManager())) throw;
+    modifier onlyRoundsManager() {
+        require(msg.sender == address(roundsManager()));
+        _;
+    }
+
+    // Only the JobsManager can call
+    modifier onlyJobsManager() {
+        require(msg.sender == address(jobsManager()));
         _;
     }
 
     /*
      * @dev BondingManager constructor. Sets a pre-existing address for the LivepeerToken contract
      * @param _token LivepeerToken contract address
      */
-    function BondingManager(address _token, uint256 _numActiveTranscoders) {
+    function BondingManager(address _registry, address _token, uint256 _numActiveTranscoders) Manager(_registry) {
```

Comment:
It is the constructor for the base contract (http://solidity.readthedocs.io/en/develop/contracts.html#arguments-for-base-constructors). So in this case since `BondingManager` inherits from `Manager` it needs to invoke the constructor for `Manager`